README.rst

```diff
@@ -5,6 +5,33 @@ Astronomical data analysis development leveraging Jupyter platform
     :target: http://www.astropy.org
     :alt: Powered by Astropy Badge
 
+About
+-----
```

Comment:
```suggestion
```

You don't need the about header, it's fine to just have the descriptions below the main header.